### PR TITLE
Fix Issue #483

### PR DESCRIPTION
**Pull request by AshutoshFreak**
Fixed issue #483: Altered 3 links within the README document to reference files in the 'main' branch instead of the 'oracle2bq-notebook' branch

### CODE REVIEW

File: notebooks/oracle2bq/README.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
 Notebook solution utilizing dataproc templates for migrating databases from Oracle to BigQuery. Notebook contains step by step process for migrating Oracle database tables to BigQuery.
 
 Refer [Setup Vertex AI - PySpark](./../generic_notebook/README.md) to setup new Jupyter notebook in vertexAI. Once the setup is done navigate to
-[dataproc-templates/python/notebooks/oracle2bq](https://github.com/GoogleCloudPlatform/dataproc-templates/tree/oracle2bq-notebook/notebooks/oracle2bq) folder and open [OracleToBigQuery_vertex_pipeline_pyspark.ipynb](https://github.com/GoogleCloudPlatform/dataproc-templates/blob/oracle2bq-notebook/notebooks/oracle2bq/OracleToBigQuery_notebook.ipynb) notebook.
+[dataproc-templates/python/notebooks/oracle2bq](https://github.com/GoogleCloudPlatform/dataproc-templates/tree/main/notebooks/oracle2bq) folder and open [OracleToBigQuery_vertex_pipeline_pyspark.ipynb](https://github.com/GoogleCloudPlatform/dataproc-templates/blob/main/notebooks/oracle2bq/OracleToBigQuery_notebook.ipynb) notebook.
```
Comment (shashank-google) [inline notebooks/oracle2bq/README.md:6]: we should use relative paths when linking to other files of the same repo eg:`[OracleToBigQuery_vertex_pipeline_pyspark.ipynb](/notebooks/oracle2bq/OracleToBigQuery_notebook.ipynb)`

**Review by shashank-google — CHANGES_REQUESTED** [review]

**Comment by AshutoshFreak** [comment]
I've fixed the relative path issue. I am following the commit conventions, but it doesn't seem to pass the tests. Does the PR name needs to follow the same convention? I'll have to close this PR and open a new one if that's the case

**Comment by vanshaj-bhatia** [comment]
> 

@AshutoshFreak Please rename the PR using the same conventions.

> @shashank-google I've fixed the relative path issue. I am following the commit conventions, but it doesn't seem to pass the tests. Does the PR name needs to follow the same convention? I'll have to close this PR and open a new one if that's the case

**Comment by ankuljain09** [comment]
@AshutoshFreak - You could close the PR and reopen to make the commit-checker bot to run again. Also, please ensure to push a new commit message that looks similar to this-
fix: Updated Readme with correct links

**Pull request closed without merging**